index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    'use strict';
+    
+console.log("========================");
+console.log("現在持っているタスク一覧");
+console.log("========================");
+
+ const daily_list = [
+ {genre: '掃除', task: '机を片付ける、'},
+ {genre: '買い物', task: '牛乳を買う、'},
+ {genre: '通勤', task: '散歩する、'},
+];
+
+
+daily_list.forEach(function(key){
```

Comment:
```suggestion
daily_list.forEach((key) => {
```
とする方が、今どきの書き方です（Web 上のサンプルコードは古い記述のものが多いので要注意です）

Reply:
次回以降は気をつけたいと思います！